### DISCUSSION TITLE
Filtered Directories

### DISCUSSION THREAD

**Discussion by diggyk** [microsoft/vscode-discussions#362] [Extension Show and Tell]
I work in a giant monorepo and I hate scrolling past all the directories I don't care about.  So I made an extension that just shows the directories I care about.  Feedback and bug reports welcome: https://marketplace.visualstudio.com/items?itemName=diggyk.fildir&ssr=false#overview

**Comment by burkeholland** [comment]
This is pretty cool! I know we have a lot of mono repo users out there. I've got Twitter rotation this week, so I'll give it a bump. Nice work!
